jar-persistence/src/main/java/com/sixsq/slipstream/persistence/User.java

```diff
@@ -949,18 +950,18 @@ private void checkNoForbiddenRoles(String roles) throws ValidationException {
         if (roles == null || roles.isEmpty()) {
             return;
         }
-        for (String role : roles.split(",")) {
+        for (String role : roles.split(" ")) {
             String trimedUppercaseRole = role.trim().toUpperCase();
             if (FORBIDDEN_ROLES.contains(trimedUppercaseRole)) {
```

Comment:
This check will probably need to be removed as the ADMIN, USER, and ANON roles will probably already be present in the claims.

Reply:
Yes, I also noticed that and wanted to fix it when running the full test with authn/z.  I'm preparing my dev env for the test at the moment.